### PR TITLE
Added integration string and unit test

### PR DESCRIPTION
**Pull request by eatwithforks**

### CODE REVIEW

File: lib/__init__.py [hunk 1]
```diff
@@ -3,4 +3,8 @@
 from lib.event import Event
 from lib.options import Options
 from lib.utility import Utility
-from lib.rsyslog import Rsyslog
\ No newline at end of file
+from lib.rsyslog import Rsyslog
+from lib.config_helper import ConfigHelper
+
+__author__ = "toolbox@cloudpassage.com"
+__version__ = "1.8"
```
Comment (ashmastaflash) [inline lib/__init__.py]: Need to add a newline at the end of the file.

File: lib/config_helper.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        ua_string = product + "/" + version
+        return ua_string
+
+    @classmethod
+    def get_product_version(cls):
+        init = open(os.path.join(os.path.dirname(__file__),
+                    "__init__.py")).read()
+        rx_compiled = re.compile(r"\s*__version__\s*=\s*\"(\S+)\"")
+        version = rx_compiled.search(init).group(1)
+        return version
```
Comment (ashmastaflash) [inline lib/config_helper.py]: Newline here, too...

Comment (eatwithforks) [inline lib/config_helper.py]: fixed

File: lib/__init__.py (hunk 1 above)
Comment (eatwithforks) [inline lib/__init__.py]: fixed

File: tests/unit/test_unit_halo.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        return session
+
+    def test_integration_string(self):
+        int_string = "integration/v1.0"
+        ua_string = "sdk/v1.1"
+        session = cloudpassage.HaloSession("", "",
+                                           integration_string=int_string,
+                                           user_agent=ua_string)
+        desired = "%s %s" % (int_string, ua_string)
+        assert desired == session.user_agent
```
Comment (ashmastaflash) [inline tests/unit/test_unit_halo.py]: Newline...

Comment (eatwithforks) [inline tests/unit/test_unit_halo.py]: fixed

**Comment by ashmastaflash** [comment]
Me gusta.

**Pull request merged**